tensorflow/contrib/android/java/org/tensorflow/contrib/android/TensorFlowInferenceInterface.java

```diff
@@ -264,6 +266,7 @@ public void close() {
     closeFeeds();
     closeFetches();
     sess.close();
+    runner.close();
```

Comment:
Does this work? I don't think `Session.Runner` has a `close` method - see https://github.com/tensorflow/tensorflow/blob/master/tensorflow/java/src/main/java/org/tensorflow/Session.java